Document/MediaManager.php

```diff
@@ -30,11 +30,13 @@ public function save($entity, $andFlush = true)
     {
         // BC compatibility for $context parameter
         if ($andFlush && is_string($andFlush)) {
+            trigger_error('Since version 2.2.7, passing context in argument 2 for '.__METHOD__.'() is deprecated and support for it will be removed in 3.0. Use '.$this->class.'::setContext() instead.', E_USER_DEPRECATED);
```

Comment:
@Soullivaneuh, [```__METHOD__```](http://php.net/manual/en/language.constants.predefined.php) already includes the QCN, so your suggestion will output something like:
```
Since version 2.2.7, passing provider name in argument 3 for Sonata\MediaBundle\Entity\MediaManager::Sonata\MediaBundle\Entity\MediaManager::save() is deprecated and support for it will be removed in 3.0. Use AppBundle\Entity\Media::setProviderName() instead.
```

BTW, the syntax I used is the same as used in Symfony codebase. Check [this deprecation message](https://github.com/symfony/symfony/blob/ae470cdf148d5b779623f6922e45160fe14e9b6f/src/Symfony/Component/Form/Extension/Csrf/CsrfProvider/CsrfTokenManagerAdapter.php#L40) by instance.